wurst/objects/units/ShopObjects.wurst

```diff
@@ -0,0 +1,36 @@
+package ShopObjects
+
+import Assets
+import UnitObjEditing
+
+import LocalObjectIDs
+
+public class ShopHeroDefinition extends HeroDefinition
```

Comment:
Create a method for this, e.g.

    public UnitDefinition.setDummyStockInterface()
        this
            ..setLumberCost(0)
            ..setFoodCost(0)
            ..setGoldCost(0)
            ..setStockStartDelay(0)
            // Expect restocks to be triggered directly.
            ..setStockReplenishInterval(999) 
            ..setStockMaximum(1)